cpp/src/Ice/SSL/SchannelTransceiverI.cpp

```diff
@@ -357,13 +353,9 @@ Schannel::TransceiverI::sslHandshake(SecBuffer* initialBuffer)
         }
     }
 
-    //
-    // Check if the requested capabilities are met.
-    //
-    // NOTE: it's important for _ctxFlags to be a data member. The context flags might not be checked immediately
-    // if the last write can't complete without blocking above. In such a case, the context flags are checked here
-    // only once the sslHandshake is called again after the write completes.
-    //
+    // Check if the requested capabilities are met. NOTE: it's important for _ctxFlags to be a data member. The context
+    // flags might not be checked immediately if the last write can't complete without blocking above. In such a case,
+    // the context flags are checked here only once the sslHandshake is called again after the write completes.
```

Comment:
`NOTE: ` should have been keep in its own line, and the `// ` to separate the paragraph was fine.